extensions/spring-web/deployment/src/main/java/io/quarkus/spring/web/deployment/SpringWebProcessor.java

```diff
@@ -338,7 +338,7 @@ public void generateExceptionMapperProviders(BeanArchiveIndexBuildItem beanArchi
         // Look for all exception classes that are annotated with @ResponseStatus
 
         IndexView index = beanArchiveIndexBuildItem.getIndex();
-        ClassOutput classOutput = new GeneratedClassGizmoAdaptor(generatedExceptionMappers, false);
+        ClassOutput classOutput = new GeneratedClassGizmoAdaptor(generatedExceptionMappers, true);
```

Comment:
Oops :).

Reply:
Yeah, that cost me a good hour or so to figure out... Paying for my own mistakes :P